e2e/test/scenarios/native-filters/sql-field-filter.cy.spec.js

```diff
@@ -53,6 +53,13 @@ describe("scenarios > filters > sql filters > field filter", () => {
       SQLFilter.getRunQueryButton().should("be.disabled");
       SQLFilter.getSaveQueryButton().should("have.attr", "disabled");
 
+      SQLFilter.getSaveQueryButton().realHover();
+      cy.get("body").within(() => {
```

Comment:
I am keeping it `body` for now because I think doing `[data-tippy-root]` is too specific for the current implementation and e2e tests should rather avoid depending on the _3rd party lib_ implementation details.

Reply:
Hint: `cy.get("body").findByText("foo");` works just fine.

Reply:
@oleggromov `cy.findByRole("tooltip")` should be used